eiffel-vocabulary/EiffelTestSuiteStartedEvent.md

```diff
@@ -52,10 +52,145 @@ __Type:__ String
 __Required:__ Yes  
 __Description:__ The URI at which the log can be retrieved.
 
+## Links
+### TERC
+__Required:__ No  
+__Legal targets:__ [EiffelTestExecutionRecipeCollectionCreatedEvent](../eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md)  
+__Multiple allowed:__ No  
+__Description:__ When declared in an EiffelTestSuiteStartedEvent, this link signifies that the test suite groups all test case executions resulting from the [EiffelTestExecutionRecipeCollectionCreatedEvent](../eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md).
```

Comment:
This when statement is a bit reduntant now,  since the description is in the EiffelTestSuiteStartedEvent